src/test/java/seedu/address/logic/commands/ClientNoteEditCommandTest.java

```diff
@@ -53,22 +53,53 @@ public void execute_validClientIdxValidNoteIdx_generatesClientNoteEditCommandSuc
         ClientNoteEditCommand clientNoteEditCommand = new ClientNoteEditCommand(clientIdx, clientNoteIdx, newEditNote);
         assertCommandSuccess(clientNoteEditCommand, model, expectedResult, expectedModel);
     }
+    // test to show untagged tag removal (in future commit) behaviour is already verified by json file
```

Comment:
Needs a new line before this comment. Also, I'm not too sure what this comment means?

Reply:
Removed the comment because the test name is descriptive enough